php/WP_CLI/Runner.php

```diff
@@ -334,6 +334,44 @@ private function _run_command() {
 	}
 
 	/**
+	 * Perform a command against a remote server over SSH
+	 */
+	private function run_ssh_command( $ssh ) {
+
+		$host = $ssh;
+		$path = '';
+		if ( false !== ( $key = stripos( $host, ':' ) ) ) {
+			$path = substr( $host, $key + 1 );
+			$host = substr( $host, 0, $key );
+		}
+
+		WP_CLI::debug( 'SSH host: ' . $host, 'bootstrap' );
+		WP_CLI::debug( 'SSH path: ' . $path, 'bootstrap' );
+
+		$is_tty = function_exists( 'posix_isatty' ) && posix_isatty( STDOUT );
+
+		$wp_binary = 'wp';
```

Comment:
Ah it looks difficult.

* Place the `~/.ssh/environment` into local machine.
* Add `PermitUserEnvironment yes` in `/etc/ssh/sshd_config` on remote server.

http://stackoverflow.com/questions/216202/why-does-an-ssh-remote-command-get-fewer-environment-variables-then-when-run-man